src/database/schema/ChatSchema.ts

```diff
@@ -197,7 +197,7 @@ class ChatSchema implements BaseDbSchema {
   }
 
   static fromWebsocketObject(json): ChatSchema {
-    let rawJson = null;
+    let rawJson: string | null = null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `let rawJson = null;` to `let rawJson: string | null = null;` is a good one as it provides type safety by explicitly stating that `rawJson` can be either a string or null. This helps prevent potential bugs and makes the code easier to understand.
```diff
-    let rawJson = null;
+    let rawJson: string | null = null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->